### DISCUSSION TITLE
Lagrange.Kritor uin/uid format

### DISCUSSION THREAD

**Discussion by KusaGumina** [LagrangeDev/Lagrange.Core#529] [Polls]
Lagrange.Kritor uin/uid format

**Comment by JustAnotherID** [comment]
作为下游使用者，需要普通用户可知的uin，同时该字段都是以str形式使用，希望可以直接获取无需转换。

**Reply by sj817** [reply]
下游+1

**Reply by 2mf8** [reply]
下游+1

**Comment by ghost** [comment]
对于lgr.kritor来说，我觉得只要uin str够了

但是对于整个kritor协议来说，我觉得必须提供uin必选和uid可选字段

所以面向单个实现lgr.kritor，我支持uin str。面向整个协议Kritor，我选择uin + optional uid
